cmd/qsctl/init.go

```diff
@@ -122,13 +123,20 @@ func initConfig() (err error) {
 
```

Comment:
If we choose to going on, we need to set `err = nil` so that it will not affect following error check